sycl/tools/sycl-ls/CMakeLists.txt

```diff
@@ -1,9 +1,16 @@
 add_executable(sycl-ls sycl-ls.cpp)
 add_dependencies(sycl-ls sycl)
 target_include_directories(sycl-ls PRIVATE "${sycl_inc_dir}")
+
+set(sycl_lib sycl)
```

Comment:
Should we define this at a top level and re-use in all linking?

Reply:
Do we have/plan to have other tools, that link sycl runtime dynamically?

Reply:
I hope so, but I'd suggest to merge existing users of "sycld", like this https://github.com/intel/llvm/blob/0e22fdd28acb26253cbf5a1048daa76cf739c309/sycl/CMakeLists.txt#L116

Reply:
I see. I agree that this is a problem, but I'm not a big fan of global variables. I think, the right way to fix this is to introduce `add_sycl_library`/`add_sycl_executable`, but that's a bit out of scope of this patch. Is it ok with you to commit this patch now to fix the immediate problem and then I will follow up with some CMake refactorings to accommodate such issues?

Reply:
Sure. I may want to add a TODO comment in this PR.